completions/gh.completion.sh

```diff
@@ -2,9 +2,9 @@
 # This script complements the completion script that ships with git.
 
 # Check that git tab completion is available
-if declare -F _git > /dev/null; then
+if _omb_util_function_exists _git > /dev/null; then
   # Duplicate and rename the 'list_all_commands' function
-  eval "$(declare -f __git_list_all_commands | \
+  eval "$(_omb_util_function_exists __git_list_all_commands | \
```

Comment:
This use `declare -f` is for extracting the definition of the function, but not to test the existence of the function, so this change should not be applied.